hack/tools/Makefile

```diff
@@ -76,6 +76,10 @@ CONVERSION_GEN := $(BIN_DIR)/conversion-gen
 $(CONVERSION_GEN): $(BIN_DIR) go.mod go.sum $(call rwildcard,.,third_party/conversion-gen/*.*)
 	go build -tags=tools -o $@ ./third_party/conversion-gen
 
+CONVERSION_VERIFIER := $(BIN_DIR)/conversion-verifier
+$(CONVERSION_VERIFIER): $(BIN_DIR) go.mod go.sum
+	go build -tags=tools -o $@ ./third_party/conversion-verifier
```

Comment:
I think we should build it from the cluster-api repo instead of mnainting our own version of the tool.

Reply:
https://github.com/kubernetes-sigs/cluster-api/tree/main/hack/tools/conversion-verifier

Reply:
Yes, you're right, I'll fix this asap. Thanks for pointing it out!

Reply:
Thanks